src/stories/TextInputWithTokens.stories.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useCallback, useState } from 'react'
+import {Meta} from '@storybook/react'
+
+import { BaseStyles, Box, ThemeProvider } from '..'
+import TextInputTokens from '../TextInputWithTokens';
+import TokenLabel from '../Token/TokenLabel';
+
+export default {
+  title: 'Prototyping/Text Input with Tokens',
+
+  decorators: [
+    Story => {
+      const [lastKey, setLastKey] = useState('none')
+      const reportKey = useCallback((event: React.KeyboardEvent<HTMLDivElement>) => {
+        setLastKey(event.key)
+      }, [])
+
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <Box onKeyDownCapture={reportKey}>
+              <Box position="absolute" right={5} top={2}>
+                Last key pressed: {lastKey}
+              </Box>
+              <Box paddingTop={5}>
+                <Story />
+              </Box>
+            </Box>
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    }
+  ]
+} as Meta
+
+const mockTokens = [
+  { text: 'zero', id: 0 },
+  { text: 'one', id: 1 },
+];
+
+export const Default = () => {
+    const [tokens, setTokens] = useState(mockTokens)
+    const onTokenRemove: (tokenId: string | number) => void = (tokenId) => {
+        setTokens(tokens.filter(token => token.id !== tokenId))
+    };
+
+    return (
+      <TextInputTokens
+          tokens={tokens}
+          onTokenRemove={onTokenRemove}
+      />
+    )
+};
+
+export const TokenSizeVariants = () => {
+  const [tokens, setTokens] = useState(mockTokens)
+  const onTokenRemove: (tokenId: string | number) => void = (tokenId) => {
+      setTokens(tokens.filter(token => token.id !== tokenId))
+  };
+
+  return (
+    <Box display="flex" flexDirection="column" sx={{'gap': '1em'}}>
+      <TextInputTokens
+          tokens={tokens}
+          onTokenRemove={onTokenRemove}
+          tokenSizeVariant="sm"
```

Comment:
Could we just call this `size` since it affects the size of the text input as well?
```suggestion
          size="sm"
```

Reply:
Good point. Agreed that we should update this prop name